.eslintrc.js

```diff
@@ -11,6 +11,17 @@ module.exports = {
     SharedArrayBuffer: 'readonly',
     createjs: false,
   },
+  overrides: [
```

Comment:
Override added specifically to target unit test files. This doesn't include the setup file, since I figured that'd help ensure that we know that they're being set in the unit test files.